examples/seismic/tutorials/03_fwi.ipynb

```diff
@@ -514,37 +498,33 @@
   {
    "cell_type": "code",
    "execution_count": 13,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Create bounds constraint\n",
     "def bound_constr(m):\n",
-    "    m[m<.1] = .08 # Maximum accepted velocity is 3.5 km/sec (true is 3 km/sec)\n",
-    "    m[m>.2] = .25 # Minimum accepted velocity is 2 km/sec (true is 2.5 km/sec)\n",
-    "    return m"
+    "    # Maximum accepted velocity is 3.5 km/sec (true is 3 km/sec)\n",
+    "    # Minimum accepted velocity is 2 km/sec (true is 2.5 km/sec)\n",
+    "    return np.clip(m, 0.08, 0.25)"
```

Comment:
The wording seems a bit confusing. Is the true 3 km/sec or 2.5 km/sec?

Reply:
True are 2.5 and 3., but' it's a bit too much cheating to put the true max/min values.

Reply:
Not what I'm saying. You wrote "True are 2.5 and 3." which is clear. "true is 3 km/sec" and "true is 2.5 km/sec" are not, because they are contradicting each other. "The true velocities are 3 and 2.5 km/sec" is what I'm asking for.

Reply:
It's not contradicting, first line is Maximum velocity with true at 3 and second is minimum velocity with true at 2.5.

Reply:
Changing this to:
    # Maximum possible 'realistic' velocity is 3.5 km/sec
    # Minimum possible 'realistic' velocity is 2 km/sec
    return np.clip(m, 1/3.5**2, 1/2**2)